docker-compose.yml

```diff
@@ -12,13 +12,16 @@ services:
     image: docker.artifactory.reform.hmcts.net/reform/draft-store-api
     environment:
       - DRAFT_STORE_DB_HOST=draft-store-database
+      - DRAFT_STORE_DB_PORT
       - DRAFT_STORE_DB_PASSWORD
       - ROOT_APPENDER
       - JSON_CONSOLE_PRETTY_PRINT
       - ROOT_LOGGING_LEVEL
       - REFORM_SERVICE_NAME
       - REFORM_TEAM
       - REFORM_ENVIRONMENT
+      - IDAM_USE_STUB
+      - S2S_USER_STUB
```

Comment:
typo

Reply:
To be removed in all occurrences as a separate change